### DISCUSSION TITLE
snmp Community string export to check_mk

### DISCUSSION THREAD

**Discussion by oliverlangkau** [kuhn-ruess/cmdbsyncer#41] [General]
Hi,
I'm trying to set the snmp community string in check_mk from cmdbsyncer.
And I don't understand how to target the community string in check_mk.
In check_mk swagger i saw the community is part of a dictionary:

  "attributes": {
      "ipaddress": "10.220.7.130",
      "snmp_community": {
        "type": "v1_v2_community",
        "community": "public"
      },

For the ip adress i create a rewrite rule to change the namen from ip (our csv iomport to cmdbsyncer) to ipaddress (check_mk attrib name). This works.
SNMP community i try to use a rewrite rule from SNMP_Community to community. Don't work.
And i try to set with "Custom CMK Attribute" rule to "tag_snmp_community:{{SNMP_community}}". Also don't work.

In both rules i had this outcome tag_type:v1_v2_community, too.

CMDB Syncer 3.8 Daily 18.10.2024
Checkmk Enterprise Edition 2.3.0p13

Can you please give me a hint what I'm doing wrong.

Best regards

oliver

**Comment by Bastian-Kuhn** [comment]
Hello Oliver,

I updated the Documentation here: https://cmdbsyncer.readthedocs.io/en/latest/checkmk/cmk_attributes/ (end of Page) how to do that. In Essence, this attribute is a dict, and you need to thread it as such. It's not a Tag. But the full example, what you need to set, you find in the docs now.

Best Regards
Bastian
